docs/howtoguides/how_to_roll_out_across_multiple_machines.rst

```diff
@@ -0,0 +1,48 @@
+.. _roll_out_multiple_machines:
+
+How to roll out your pro token across multiple machines
+=======================================================
+
+This guide will show you how to roll out your pro token across multiple machines using cloud-init for configuration.
+
+Configure using Cloud-init
+--------------------------
+
+The next step is to usea cloud-init to configure your machines with the pro token.
+
+To do this, a cloud-init configuratioin file is required. Create a cloud-init.yaml file with the following content:
```

Comment:
This is probably step 2 - I wouldn't phrase it as "create a cloud-init.yaml file", because the typical reader of our docs is unlikely to know what that is. Better to tell them to create a file called cloud-init.yaml, then to copy and paste the contents of the code block.